dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -771,6 +771,14 @@ ERROR_UNKNOWN_OSLIST_VALUE=CWWKF1387E: The value {0} is not a valid entry for th
 ERROR_UNKNOWN_OSLIST_VALUE.explanation=This value will be ignored as it is not a recognized operating system name.
 ERROR_UNKNOWN_OSLIST_VALUE.useraction=Please modify or remove this value from the feature manifest file.
 
+ERROR_ESA_NOT_FOUND=CWWKF1388E: The specified ESA file did not contain any features.
+ERROR_ESA_NOT_FOUND.explanation=A feature will not be installed from the ESA file.
+ERROR_ESA_NOT_FOUND.useraction=Ensure that the individual ESA file provided is valid and contains a valid feature.
+
+ERROR_SINGLE_REPO_CONNECTION_FAILED=CWWKF1389E: Unable to establish a single file repository connection for the single JSON.
```

Comment:
[SingleFileRepositoryConnection](https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.repository/src/com/ibm/ws/repository/connections/SingleFileRepositoryConnection.java) throws IOException if the json already exists or there is a problem creating the file.  We can assume it doesn't already exist since InstallKernelMap line 810 checked for that, so the issue would be a problem creating the file.  So the message can be like:

```
ERROR_SINGLE_REPO_CONNECTION_FAILED=CWWKF1389E: Unable to create a JSON file at {0} for the ESA file {1}.
ERROR_SINGLE_REPO_CONNECTION_FAILED.explanation=The feature in the ESA file will not be installed.
ERROR_SINGLE_REPO_CONNECTION_FAILED.useraction=Ensure that write access is available for the directory where the JSON file will be created.
```